_sources/Dictionaries/Exercises.rst

```diff
@@ -228,14 +228,15 @@ Exercises
 
     .. sourcecode:: python
 
-        alphabet = 'abcdefghijklmnopqrstuvwxyz'
-        values = {}
-        for i in range(len(alphabet)):
-            values[alphabet[i]] = i+1;
-        answer  = 0
-        for c in s:
-            answer += values[c]
-        return answer
+		def foo(s):
```

Comment:
All our examples and other code use 4 space indentation.  Please change this to match.